src/reducers/index.js

```diff
@@ -1,9 +1,11 @@
 /* eslint-disable import/extensions */
```

Comment:
With the ESLint inline comment, you are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.